# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/components/contextualidentity/test/browser/browser_saveLink.js
# Commit: 3ce54e39ebcd
# Full Hash: 3ce54e39ebcdf069615571d1805af75ad969f006
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2020-06-03 04:20:58
# Regressor Bug: 1638358
# File Overlap Count: 1
# Description:
#   Bug 1638358 - Cookie Schemeful Same-Site - part 5 - schemeful comparison, r=mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D75629
# ==============================================================================

diff -r dff1c1aca50c -r 3ce54e39ebcd browser/components/contextualidentity/test/browser/browser_saveLink.js
--- a/browser/components/contextualidentity/test/browser/browser_saveLink.js	Tue Jun 02 13:49:45 2020 +0000
+++ b/browser/components/contextualidentity/test/browser/browser_saveLink.js	Tue Jun 02 13:50:12 2020 +0000
@@ -13,7 +13,13 @@
 
   // make sure userContext is enabled.
   await SpecialPowers.pushPrefEnv({
-    set: [["privacy.userContext.enabled", true]],
+    set: [
+      ["privacy.userContext.enabled", true],
+      // This test does a redirect from https to http and it checks the
+      // cookies. This is incompatible with the cookie sameSite schemeful
+      // feature and we need to disable it.
+      ["network.cookie.sameSite.schemeful", false],
+    ],
   });
 });
 